sunflower-land/sunflower-land
# [FEAT] Default a Bumpkin if missing

# Description

Allows legacy players to return to Sunflower Land 🤗 

If they are missing a Bumpkin, a default one will be given to them.

**Includes**

- Removes missing bumpkin code
- Cleans up other related code

**TODO**

- Deploy mainnet contract

## How to test?

Update your `.env` -> `VITE_ACCOUNT_MINTER_CONTRACT=0x891a1910fCf5Fae3e91DECE231806fae720b7743`

1. Connect to local API
2. Delete your Bumpkin from the DB
3. Reload & voila....new Bumpkin!

There was a lot of cleaned up code that is no longer used. It would be useful to verify other Bumpkin related flows to ensure working smoothly:

- Mint a farm (no longer mints a Bumpkin = much cheaper!)
- Dequipping Bumpkin
- Depositing + Withdrawing Wearables
